### PR TITLE
Readme for contributing

### PR DESCRIPTION
**Pull request by shramee**
@enitrat Does this look good? Anything we want to add/remove?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,15 +15,12 @@
...
 
-0. Make sure you have Rust and Cargo installed with the `default` toolchain. With rustup,
-
-   - `curl https://sh.rustup.rs -sSf | sh -s`
-   - Or `Customise installation` then `Default toolchain?` set to `stable` and everything else empty.
-
-1. Run `git clone https://github.com/shramee/starklings-cairo1.git` to clone the repo
-2. Run `cd starklings-cairo1`
+1. Make sure you have Rust and Cargo installed with the `default` toolchain.  
+   With rustup `curl https://sh.rustup.rs -sSf | sh -s`
```
Comment (enitrat) [inline README.md:19]: This is technically not with rustup, but to install rustup

**Review by enitrat — APPROVED** [review]
Lgtm

File: README.md (hunk 1 above)
Comment (shramee) [reply README.md:19]: AAh, woops 🤦

Comment (shramee) [reply README.md:19]: Ohh, but it also runs rustup?

**Pull request merged**